.prettierrc.yml

```diff
@@ -0,0 +1,3 @@
+arrowParens: always
+bracketSpacing: false
```

Comment:
While we're here, I would _love_ to remove those 2 overrides. I know the first it is going to be a tough sell, but I'm still trying :D

Reply:
What's your reasoning for not always having arrow parens?

Reply:
For me it's mainly a readability / unencessity thing, but by no means am I trying to start a debate or anything. I was simply wondering if that was something we were open to change or not, totally okay if not. To be fair, Prettier makes this a lot less worthy of debating anyway (at least at write time), because one can type without arrow parens, and they will be added on save.

What do you think about `bracketSpacing` however? It seems to me that having spacing is a lot more standard, and also I find `{foo: "bar"}` generally harder to read than `{ foo: "bar" }` (of course not in a 6-char example, but when browsing through a bigger codebase).

Reply:
So, I'm totally not bothered either way snot bracket spacing. I personally would not write code like that but I don't object to it and I understand why people like the extra spaces. 

However, for the same reason as you like the spacing, I like always having the brackets in arrow functions. 

1. It makes it more visibly obvious where the parameter is
2. It means there's 1 style for all types of arrow functions (rather than 1 parameter ones looking different)
3. It means if we add an extra parameter to the function, we don't have to add in the parens at that point

It just looks nicer imo. 

But yeah, I don't mind the bracket spacing. But this is aside from the conversation about whether we should use prettier. I'm not awake enough for that discussion today. I'll chat about it tomorrow.

Reply:
> I like always having the brackets in arrow functions.

Another point in favor of parens that I forget is when using TypeScript and explicitly typing the argument (which, granted, is fairly rare):

```ts
[1, 2, 3].map((x: number) => x * 2);
```

> But this is aside from the conversation about whether we should use prettier.

You're right, I shouldn't have brought this up so soon, that was premature, my bad.
Thanks for bringing up good points :)